src/cli.rs

```diff
@@ -25,6 +27,23 @@ pub enum Cli {
         #[arg(long)]
         commit: bool,
 
+        /// Prefix or format the commit hash used in notifications.
+        ///
+        /// In order to have some URL in the notification containing the commit hash it needs to be placed inside an URL.
```

Comment:
Hard to understand, can this be reworded?

Reply:
any suggestions? tldr:

```rs
if template.contains("{commit}") {
  template.replace("{commit}", commit)
} else {
  format!("{template}{commit}")
}
```

Reply:
I want to click on it and see a browser version.

Use case: Smartphone, change happens, click on it, view the diff.

`website-stalker` only knows the commit hash which is not clickable → add the relevant parts to have clickable URL.

Currently, this is done via the `notification_template`:

https://github.com/EdJoPaTo/website-stalker-example/blob/ff94391c10a2faf0dc02c8db2b973f972eb763aa/website-stalker.yaml#L13-L15

Reply:
AHH! That makes sense. Okay, then the wording here is not clear. I know this not a manpage, but maybe just insert an example?

Maybe this could work?
`To have a direct link to the change(s) inside the notification, provide an URL to the git instance and append the commit.`